com.zen.browser.flatpak.yml

```diff
@@ -0,0 +1,66 @@
+app-id: com.zen.browser
```

Comment:
this app-id is invalid as the domain doesn't look to be controlled by upstream developers, contra to what you acknowledged.

Reply:
I didnt know it had to be like the domain, sorry. It's currently in https://get-zen.vercel.app so I didnt want to get into any trouble with vercel

Reply:
so you answered yes to a question without reading it? The checklist isn't a CAPTCHA.

Reply:
can it be changed later though?

Reply:
If you are using this as the appid when building or in code, you have to change. If you don't own a domain, use `io.github.zen_browser.zen`. 

You need to create a repo named by the last component `zen` here too https://github.com/zen-browser otherwise it will fail a check

Reply:
> can it be changed later though?

No

Reply:
NO. It must be valid for submitting. If you read the documentation, it will tell you you can use code hosting related app-id like with github. We don't require a domain. We just require you don't steal one you have no control of.